assets/controller.yaml

```diff
@@ -33,6 +33,7 @@ spec:
             - --endpoint=$(CSI_ENDPOINT)
             - --logtostderr
             - --v=${LOG_LEVEL}
+            - --extra-labels=${CLUSTER_ID}=owned
```

Comment:
Do you need to add pass in [extra-create-metadata](https://github.com/kubernetes-csi/external-provisioner/blob/4280d5747a58e8d12430745cc1a42eb8fa0d19f0/cmd/csi-provisioner/csi-provisioner.go#L77) to csi-provisioner as well?

Reply:
No, I don't really care about PV/PVC here. It's the label on PD i need to be created, ie. the disk in GCE not the Kubernetes object.